src/servers/src/http/opentsdb.rs

```diff
@@ -75,17 +79,23 @@ pub enum OpentsdbPutResponse {
 #[axum_macros::debug_handler]
 pub async fn put(
     State(opentsdb_handler): State<OpentsdbProtocolHandlerRef>,
-    Query(params): Query<HashMap<String, String>>,
+    Query(mut params): Query<HashMap<String, String>>,
     RawBody(body): RawBody,
 ) -> Result<(HttpStatusCode, Json<OpentsdbPutResponse>)> {
     let summary = params.contains_key("summary");
     let details = params.contains_key("details");
 
+    let db = params
+        .remove("db")
+        .unwrap_or_else(|| DEFAULT_SCHEMA_NAME.to_string());
```

Comment:
we can avoid to_string by:

    let db = params
        .get("db")
        .map(|v| v.as_str())
        .unwrap_or(DEFAULT_SCHEMA_NAME);